pkg/acsengine/const.go

```diff
@@ -78,9 +78,10 @@ const (
 	DCOSPublicAgent DCOSNodeType = "DCOSPublicAgent"
 )
 
-// KubeImages represents Docker images used for Kubernetes components based on Kubernetes version
+// KubeImages represents Docker images used for Kubernetes components based on Kubernetes version hint
+// version hint here is the major and minor part of the version. For instance, version 1.6.6 has version hint 1.6
 var KubeImages = map[string]map[string]string{
-	api.Kubernetes171: {
+	api.KubernetesVersionHint17: {
```

Comment:
This is the place, when patched version is bumped up, the value of this map need to be updated. Let me look at ps1 file. I may missed it.